configs/ait/config/config.yaml

```diff
@@ -29,6 +29,72 @@ default:
 
     emitdefaults:
         filename: emitdefaults.yaml
+    # Adding AIT-DSN Plugin        
+    dsn:
+        sle:
+            initiator_id: LSE
+            # password only matters if we're doing auth and can stay as 'pw'
+            password: pw
+            responder_id: SSC
+            # peer password only matters if we're doing auth and can stay as 'pw'
+            peer_password: pw
+            downlink_frame_type: TMTransFrame
+            heartbeat: 25
+            deadfactor: 5
+            buffer_size: 256000
+            responder_port: 'default'
+            rcf:
+                inst_id: sagr=LSE-SSC.spack=Test.rsl-fg=1.rcf=onlc2
+                hostnames:
+                    - example.hostname.1
+                    - example.hostname.2
```

Comment:
Have we looked into what these do? Is there a value we should be injecting based on their deployment?

Reply:
same for below sections (`raf`, and `fcltu`)

Reply:
These are different standards for space link extensions. Some docs (here)[https://public.ccsds.org/Pubs/911x2b3.pdf] -- I imagine this is what the person using the guide would specify?

Reply:
Need to document in the deployment guide that these defaults are for reference and are not functional - all the SLE configuration will be left to the user as a post-deployment step